app/Models/Answer.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Answer extends Model
+{
+    public function contents()
+    {
+        return $this->morphMany(Content::class, 'contentable');
```

Comment:
mối quan hệ này là gì ?

Reply:
Quan hệ này là quan hệ many-to-one (Một `Answer` có nhiều `Content`).
`Content` ở đây là Polymorphic (đa hình), có một `Answer` hoặc một đối tượng khác (Ở đây là `Question`).